draft-ietf-httpbis-digest-headers.md

```diff
@@ -282,11 +285,11 @@ Representation-Digest: \
   iYllu7BNNyealdVLvRwEmTHWXvJwew==:
 ~~~
 
-A recipient MAY ignore any or all of members of `Representation-Digest`.
+A recipient MAY ignore any or all `dict-member`s.
```

Comment:
@LPardue we could replace `dict-member`s with just "members" or "digests" (which is already used here https://github.com/httpwg/http-extensions/pull/1973/files#diff-de34152d297a9eb29b85b24962949ba2e309478a5c32e1d94fd4c3ee27b4584dL271 )

Reply:
"digest" is actually mentioned in Notational Conventions in relation to the values that we have in the fields... let's use it!